frontend/src/app/pages/desktop.userProfilePage.jsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+// eslint-disable-next-line import/order
+import { useSelector } from 'react-redux';
+
+import './styles/desktop.userProfilePage.scss';
+
+const UserProfile = (): React.Node => {
+    const user = useSelector(state => state.user.name);
+    const email = useSelector(state => state.user.username);
+    return (
+        <div className="row fix justify-content-center mt-3 w-100">
+            <img src="https://st4.depositphotos.com/1012074/20946/v/450/depositphotos_209469984-stock-illustration-flat-isolated-vector-illustration-icon.jpg" />
+            <div className="row fullWidth justify-content-center mt-3">
+                <div className="box">
+                    <div className="card-header text-center">
+                        <h2>My Profile</h2>
```

Comment:
Can you make this more generic like "X's Profile" depending on the username?

Reply:
Done